ramls/loan.json

```diff
@@ -21,6 +21,11 @@
       "description": "ID of the item lent to the patron",
       "type": "string"
     },
+    "itemEffectiveLocationAtCheckOut": {
+      "description": "The effective location, at the time of checkout, of the item loaned to the patron. This is a UUID that refers to /locations/{id}.",
+      "type": "string",
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
FOLIO now [defines|https://dev.folio.org/guides/uuids/] `^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[1-5][a-fA-F0-9]{3}-[89abAB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$` as the standard UUID regular expression (in some projects, this has been defined as a reusable type). Please change this to use the newer standard.